static/src/javascripts/projects/common/modules/experiments/tests/contributions-epic-brexit-supreme.js

```diff
@@ -39,7 +39,7 @@ define([
         this.author = 'Phil Wills';
         this.description = 'Appeal linked to the Brexit appeal in the Supreme Court';
         this.showForSensitive = false;
-        this.audience = 1;
+        this.audience = 0.7;
```

Comment:
This should probably go up to where the next test starts, which is currently 0.85 (although I've suggested it should be 0.88 in another comment)